packages/analytics-js/__tests__/components/capabilitiesManager/CapabilitiesManager.test.ts

```
@@ -32,17 +33,19 @@
+  describe('init', () => {
     beforeEach(() => {
-      capabilitiesManager = new CapabilitiesManager(defaultErrorHandler, mockLogger);
+      capabilitiesManager = new CapabilitiesManager(
+        defaultHttpClient,
+        defaultErrorHandler,
+        defaultLogger,
+      );
     });
 
     afterEach(() => {
+      POLYFILL_URL = 'https://somevalid.polyfill.url';
```

Biome

[error] The imported variable POLYFILL_URL is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)